documentation/pom.xml

```diff
@@ -39,7 +39,7 @@
         <!-- Skip artifact deployment -->
         <maven.deploy.skip>true</maven.deploy.skip>
         <gpg.skip>true</gpg.skip>
-        <surefire.jvm.args.additional>-Duser.language=en</surefire.jvm.args.additional>
+        <surefire.jvm.args.additional>-Duser.language=en -Duser.country=EN</surefire.jvm.args.additional>
```

Comment:
I don't think `EN` is a valid country for English or it is an obscure one. Maybe you should make it `US` instead?

Other than that, I agree we should merge this.

Reply:
Ideally, when you are done with the `US` change, please squash your comments and prefix the commit message with `HV-1968 `.

Thanks!

Reply:
done :-)